docs/tutorials/dataflows/chatbot.rst

```diff
@@ -22,32 +22,50 @@ messages which are directed to our bot.
     `examples/dataflow/chatbot <https://github.com/intel/dffml/blob/master/examples/dataflow/chatbot>`_
     directory of the DFFML source code.
 
+You'll need to install `aiohttp <https://github.com/aio-libs/aiohttp>`_
+and ``dffml-model-scikit`` (The model used for prediction).
+
+.. code-block:: console
+
+    $ pip install aiohttp dffml-model-scikit
+
 We'll write the operations for this dataflow in operations.py
 
+Adding necessary imports and defining ``Definitions`` for operation
+inputs.
+
 .. literalinclude:: /../examples/dataflow/chatbot/operations.py
```

Comment:
Whenever we have multiple files we should explicitly call out their filename every time (I've noticed users tend to get confused)

```suggestion
**operations.py**

.. literalinclude:: /../examples/dataflow/chatbot/operations.py
```